globals.go

```diff
@@ -120,6 +121,9 @@ var (
 	// be thread safe and non-blocking.
 	ErrorHandler func(err error)
 
+	// ExitFunc is the fucntion called to exit the application, defaults to `os.Exit()`
+	ExitFunc exitFunc = os.Exit
```

Comment:
You are not using `exitFunc` except here and you didn't add a comment.

So I don't think you use this type, so I would like to suggest this

```suggestion
	ExitFunc func(int) = os.Exit
```

So I'm also suggesting to delete the exitFunc type [here](https://github.com/rs/zerolog/pull/717#discussion_r2006417117)